### ISSUE TITLE
libicns broken by 2.0.1

### ISSUE THREAD

**Issue by ilovezfs** [mdadams/jasper#98]
7e452109a3e7eb52685d48c0b293a551c1535ea6 (based on git bisect) has broken the build of libicns.

```
==> make install
Making install in src
/bin/sh ../libtool --tag=CC   --mode=compile /usr/bin/clang -DHAVE_CONFIG_H -I. -I..   -F/usr/local/Frameworks -Wall -Os -w -pipe -march=native -mmacosx-version-min=10.11 -arch x86_64 -arch i386 -c -o icns_debug.lo icns_debug.c
libtool: compile:  /usr/bin/clang -DHAVE_CONFIG_H -I. -I.. -F/usr/local/Frameworks -Wall -Os -w -pipe -march=native -mmacosx-version-min=10.11 -arch x86_64 -arch i386 -c icns_debug.c  -fno-common -DPIC -o .libs/icns_debug.o
In file included from icns_debug.c:27:
In file included from ./icns_internals.h:53:
In file included from /usr/local/include/jasper/jasper.h:66:
/usr/local/include/jasper/jas_config.h:6:10: fatal error: 'jasper/jas_dll.h' file not found
#include <jasper/jas_dll.h>
         ^
1 error generated.
make[1]: *** [icns_debug.lo] Error 1
make: *** [install-recursive] Error 1
/usr/local/Homebrew/Library/Homebrew/debrew.rb:11:in `raise'
BuildError: Failed executing: make install
1. raise
2. ignore
3. backtrace
4. irb
5. shell
```

This is libicns-0.8.1 on macOS 10.11.

**Comment by mdadams** [comment]
I'm sorry about this.  The new jas_dll.h header file did not previously get added to the appropriate CMakeLists.txt file.  This should now be fixed in commit 8bdcbe042cc722f19a71a12010e1efadacc690a8.  If you have any further problems, please let me know.

**Comment by ilovezfs** [comment]
@mdadams No need to apologize. I'm one of the Homebrew maintainers just reflecting our CI results back to you: https://bot.brew.sh/job/Homebrew%20Core/12356/
